src/content-feature.js

```diff
@@ -235,6 +235,33 @@ export default class ContentFeature {
     load (args) {
     }
 
+    /**
+     * This is a wrapper to ensure all calls are typed via schemas.
+     * @type {(...args: any[]) => any}
+     */
+    notify (...args) {
+        const [name, params] = args
+        this.messaging.notify(name, params)
+    }
+
+    /**
+     * This is a wrapper to ensure all calls are typed via schemas.
+     * @type {(...args: any[]) => Promise<any>}
+     */
+    request (...args) {
+        const [name, params] = args
+        return this.messaging.request(name, params)
```

Comment:
Can we fast follow to make `this.messaging `this internal either by making it private # or at least have prefix notation.

Reply:
I think we should mark this.messaging as @deprecated in this PR. That way nothing changes but people start to see info in their editors about it.